codegen-client/src/main/kotlin/software/amazon/smithy/rust/codegen/client/smithy/endpoint/generators/EndpointParamsGenerator.kt

```diff
@@ -255,10 +255,11 @@ internal class EndpointParamsGenerator(private val parameters: Parameters) {
                 "ParamsError" to paramsError(),
             ) {
                 val params = writable {
+                    Attribute.Custom("allow(clippy::unnecessary_lazy_evaluations)").render(this)
```

Comment:
Why is this needed? 🤔

Reply:
when it's a boolean value, the generated code isn't a function invocation